packages/client/test/unit/Publisher.test.ts

```diff
@@ -79,7 +76,7 @@ describe('Publisher', () => {
                 sequenceNumber: 0,
                 streamId: STREAM_ID,
                 streamPartition: DEFAULT_PARTITION,
-                timestamp: TIMESTAMP
+                timestamp: expect.any(Number)
```

Comment:
I would the make the assertion stricter, e.g. store `Date.now()` before calling `client.publish` and then use `expect.toBeBetween(storedDate, Date.now() + 1)`.

https://github.com/jest-community/jest-extended#tobebetweenstartdate-enddate